gr-blocks/python/blocks/qa_message_debug.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2021 Free Software Foundation, Inc.
```

Comment:
You are also free to assign copyright to yourself or your organization since we are no longer tied to FSF.

Reply:
You actually need to assign it to yourself as there's paperwork that would need to be completed for you to assign it to the FSF. Thanks for the test!

Reply:
Hey, thank you for your comments. It took me a while to find a file which is not licenced to the FSF :D (would it be a good idea to include a standard copyright header to the [coding guidelines](https://github.com/gnuradio/greps/blob/master/grep-0001-coding-guidelines.md)? (could open a pr there if yes) 

also would it be more preferable if the entire block is changed to the shorter verion 

```
/* -*- c++ -*- */
/*
 * Copyright 2021 Cesar Martinez.
 *
 * This file is part of GNU Radio
 *
 * SPDX-License-Identifier: GPL-3.0-or-later

 *
 *
 */
```
which I found in the `gr-digital/include/gnuradio/digital/crc16_async_bb.h` file? (or the lengthy test as it is with my name in it instead)